cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -69,6 +73,192 @@ Usage: cardano-cli governance create-genesis-key-delegation-certificate
+
+Usage: cardano-cli governance action create-new-committee --out-file FILE
+
+  Create a new-committee action
+
+Usage: cardano-cli governance action create-constitution-update --out-file FILE
+
+  Create a constitution-update action
+
+Usage: cardano-cli governance action create-hard-fork-initiation --out-file FILE
+
+  Create a hard-fork-initiation action
+
+Usage: cardano-cli governance action create-protocol-parameter-update 
+            [--protocol-major-version NATURAL --protocol-minor-version NATURAL]
+            [--decentralization-parameter RATIONAL]
+            [--extra-entropy HEX | --reset-extra-entropy]
+            [--max-block-header-size NATURAL]
+            [--max-block-body-size NATURAL]
+            [--max-tx-size NATURAL]
+            [--min-fee-constant LOVELACE]
+            [--min-fee-linear LOVELACE]
+            [--min-utxo-value NATURAL]
+            [--key-reg-deposit-amt NATURAL]
+            [--pool-reg-deposit NATURAL]
+            [--min-pool-cost NATURAL]
+            [--pool-retirement-epoch-boundary EPOCH_BOUNDARY]
+            [--number-of-pools NATURAL]
+            [--pool-influence RATIONAL]
+            [--monetary-expansion RATIONAL]
+            [--treasury-expansion RATIONAL]
+            [--utxo-cost-per-word LOVELACE]
+            [--price-execution-steps RATIONAL --price-execution-memory RATIONAL]
+            [--max-tx-execution-units (INT, INT)]
+            [--max-block-execution-units (INT, INT)]
+            [--max-value-size INT]
+            [--collateral-percent INT]
+            [--max-collateral-inputs INT]
+            [--utxo-cost-per-byte LOVELACE]
+            --out-file FILE
+
+  Create an protocol-parameter-update action
+
+Usage: cardano-cli governance action create-treasury-withdrawal --out-file FILE
+
+  Create an treasury-withdrawal action
+
+Usage: cardano-cli governance action create-info --metadata-url URL
+            --metadata-hash HASH
+            --out-file FILE
+
+  Create an info action
+
+Usage: cardano-cli governance action view --action-file FILE
+
+  View an action
+
+Usage: cardano-cli governance action query --socket-path SOCKET_PATH
+            (--mainnet | --testnet-magic NATURAL)
+            --action-id STRING
+            [--out-file FILE]
+
+  Query an on-chain action
+
+Usage: cardano-cli governance committee 
+            ( key-gen-cold
+            | key-gen-hot
+            | key-hash
+            | authorise-hot-key
+            | resign-hot-key
+            )
+
+  Constitutional Committee operation commands
+
+Usage: cardano-cli governance committee key-gen-cold 
+            --cold-verification-key-file FILE
+            --cold-signing-key-file FILE
+
+  Create a cold key pair for a Constitutional Committee Memberkey and a new
+  certificate issue counter
+
+Usage: cardano-cli governance committee key-gen-hot --verification-key-file FILE
+            --signing-key-file FILE
+            --operational-certificate-issue-counter-file FILE
```

Comment:
Please delete  `--operational-certificate-issue-counter-file FILE `

We need no operational certificate counter file.